internal/testutil/mocknet.go

```diff
@@ -180,6 +184,36 @@ func (tn *TestNode) CreateRandomFile(t testing.TB, size int) string {
 	return file.Name()
 }
 
+func (tn *TestNode) CreateRandomBlock(t testing.TB, bs bstore.Blockstore) *blocks.BasicBlock {
```

Comment:
What was wrong with the block generate utility?

Reply:
If you are talking about `blockGen.Next()`, it's because it just generates a simple integer encoded into a block, while I needed a node encoded into a block in a cbor format: 
```json
{
"level":"error",
"error":"unable to load link: cbor input should be of type map","time":"2021-06-29T10:26:59+02:00",
"message":"failed to tag ref QmTTA2daxGqo5denp6SwLzzkLJm3fuisYEi9CoWsuHpzfb for eviction"
}
```